src/tests/bitsets/test_stdlib_bitset_large.f90

```diff
@@ -0,0 +1,1476 @@
+            error stop procedure // ' failed to interpret bitstring_0 ' // &
+                '// bitstring_0 value properly.'
+        else if ( set10 % any() ) then
+            error stop procedure // ' failed to interpret bitstring_0 ' // &
+                '// bitstring_0 value properly.'
+        else
+            write(*,*) 'from_string transferred bitstring_0//bitstring_0' // &
+                ' properly into set10'
+        end if
+
+        call set1 % from_string( bitstring_all )
+        if ( bits(set1) /= 33 ) then
+            error stop procedure // ' from_string failed to interpret ' // &
+                'bitstring_all size properly.'
+        else if ( set1 % none() ) then
+            error stop procedure // ' failed to interpret bitstring_all ' // &
+                'value properly.'
+        else if ( .not. set1 % any() ) then
+            error stop procedure // ' failed to interpret bitstring_all ' // &
+                'value properly.'
+        else if ( .not. set1 % all() ) then
+            error stop procedure // ' failed to interpret bitstring_all ' // &
+                'value properly.'
+        else
+            write(*,*) 'from_string transferred bitstring_1 properly into set1'
+        end if
+
+        call set11 % from_string( bitstring_all // bitstring_all  )
+        if ( bits(set11) /= 66 ) then
+            error stop procedure // ' from_string failed to interpret ' // &
+                'bitstring_all // bitstring_all size properly.'
+        else if ( set11 % none() ) then
+            error stop procedure // ' failed to interpret bitstring_all ' // &
+                '// bitstring_all value properly.'
+        else if ( .not. set11 % any() ) then
+            error stop procedure // ' failed to interpret bitstring_all ' // &
+                '// bitstring_all value properly.'
+        else if ( .not. set11 % all() ) then
+            error stop procedure // ' failed to interpret bitstring_all ' // &
+                '// bitstring_all value properly.'
+        else
+            write(*,*) 'from_string transferred bitstring_all // ' // &
+                'bitstring_all properly into set11'
+        end if
+
+        call set3 % read_bitset( bitstring_0, status )
+        if ( status /= success ) then
+            write(*,*) 'read_bitset_string failed with bitstring_0 as expected.'
+        end if
+
+        call set13 % read_bitset( bitstring_0 // bitstring_0, status )
+        if ( status /= success ) then
+            write(*,*) 'read_bitset_string failed with bitstring_0 ' // &
+                '// bitstring_0 as expected.'
+        end if
+
+        call set3 % read_bitset( 's33b' // bitstring_0, status )
+        if ( bits(set3) /= 33 ) then
+            error stop procedure // ' read_bitset_string failed to ' // &
+                'interpret "s33b" // bitstring_0 size properly.'
+        else if ( .not. set3 % none() ) then
+            error stop procedure // ' failed to interpret "s33b" // ' // &
```

Comment:
```suggestion
            error stop procedure // ' failed to interpret "s33b" ' // &
```

Reply:
Creating a string about concatenation looks odd, but is correct.